x-pack/plugins/security_solution/server/endpoint/lib/artifacts/lists.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createHash } from 'crypto';
+import { validate } from '../../../../common/validate';
+
+import {
+  Entry,
+  EntryNested,
+  EntryMatch,
+  EntryMatchAny,
+} from '../../../../../lists/common/schemas/types/entries';
+import { FoundExceptionListItemSchema } from '../../../../../lists/common/schemas/response/found_exception_list_item_schema';
+import { ExceptionListClient } from '../../../../../lists/server';
+import {
+  InternalArtifactSchema,
+  TranslatedEntry,
+  TranslatedEntryMatch,
+  TranslatedEntryMatchAny,
+  TranslatedEntryNested,
+  WrappedTranslatedExceptionList,
+  wrappedExceptionList,
+} from '../../schemas';
+import { ArtifactConstants } from './common';
+
+export async function buildArtifact(
```

Comment:
I don't think this needs to be `async`

Reply:
Yeah, we forgot to remove async when we removed the lzma compression this morning.